src/analytics.js

```diff
@@ -41,3 +41,20 @@ export function analyticsForOrNull(window) {
           >} */ (getElementServiceIfAvailable(
                 window, 'amp-analytics-instrumentation', 'amp-analytics')));
 };
+
+/**
+ * Helper method to trigger analytics event if amp-analytics is available.
+ * @param {!Window} window
+ * @param {string} eventType
+ * @param {!Object<string, string>=} opt_vars A map of vars and their values.
+ * @returns {!Promise<boolean>} Whether the event was sent or not.
+ */
+export function triggerAnalyticsEvent(window, eventType, opt_vars) {
+  return analyticsForOrNull(window).then(analytics => {
+    if (!analytics) {
+      return false;
```

Comment:
i guess we should also be doing some kind of logging here,  so that we know if we are triggering before analytics is available.